scripts/promote.sh

```diff
@@ -7,7 +7,6 @@ packages=(
   @patternfly/react-core
   @patternfly/react-icons
   @patternfly/react-inline-edit-extension
-  @patternfly/react-log-viewer
```

Comment:
On my catalog view removal PR @nicolethoen actually requested that I leave it in the promotion script IIRC.